.travis.yml

```diff
@@ -2,15 +2,15 @@ language : scala
 
 scala:
   - 2.11.12
-  - 2.12.4
+  - 2.12.8
 
 cache:
   directories:
   - $HOME/.ivy2
   - $HOME/.sbt
 
 jdk:
-  - oraclejdk8
```

Comment:
Why do we need to change the jdk? is it having issues with scala 2.12.8 ?

Reply:
No reason, I just don’t like using Oracle given their destructive company activities. I’m using the OpenJDK since their the most open source friendly. If you like I can revert back

Reply:
Both JDKs work absolutely fine with 2.12.8